addon/utils/group-utils.js

```diff
@@ -80,7 +80,7 @@ export function filterOptions(options, text, matcher, skipDisabled = false) {
     if (isGroup(entry)) {
       let suboptions = filterOptions(get(entry, 'options'), text, matcher, skipDisabled);
       if (get(suboptions, 'length') > 0) {
-        opts.push({ groupName: entry.groupName, options: suboptions });
```

Comment:
Without it the disabled property that is present on the options that are passed in is lost so opt.disabled for a group in the template is undefined.